larreco/QuadVtx/HeatMap.h

```diff
@@ -0,0 +1,35 @@
+#ifndef LARRECO_QUADVTX_HEATMAP_H
+#define LARRECO_QUADVTX_HEATMAP_H
+
+#include <vector>
+#include <memory>
+
+class TH2F;
```

Comment:
`std::unique_ptr<TH2F>` requires the full type specification of TH2F.  Suggest `#include "TH2F.h"` instead.

Reply:
pretty sure TH2.h explicitly defines all the TH2* types (yes ROOT...), but Done